sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -49,11 +49,13 @@ object LocalRelation {
  *
  * @param data The local collection holding the data. It doesn't need to be sent to executors
  *             and then doesn't need to be serializable.
+ * @param fromCommand Indicates whether the relation data is result of [[Command]]
+ * @param isStreaming Indicates whether this relation has data from a streaming source
  */
 case class LocalRelation(
     output: Seq[Attribute],
     data: Seq[InternalRow] = Nil,
-    // Indicates whether this relation has data from a streaming source.
+    fromCommand: Boolean = false,
```

Comment:
it seems safer to put the new parameter at the end.

Reply:
Ok, will fix it. 
I thought we should put the override members at the end.